src/components/Navigation/components/UserMenu/UserMenu.tsx

```diff
@@ -29,155 +21,52 @@ export interface Props {
 }
 
 interface State {
-  userMenuExpanded?: boolean;
+  open: boolean;
 }
 
-export default class UserMenu extends React.PureComponent<Props, State> {
-  state: State = {
-    userMenuExpanded: false,
+/** @deprecated Use <TopBar.UserMenu /> instead. */
```

Comment:
Same thing here, do we parse and display anywhere. If not, where and how would you expect to see this in the style guide? Feel free to bring the conversation to the [deprecation strategy doc](https://docs.google.com/document/d/1BitDY3l2IuK9Q7mHGUK-S3DBnV5o0xPAXZDtMom5Q4A/edit#).